sequence_prediction/sequence_generator/predict.py

```diff
@@ -324,7 +325,7 @@ def accuracy(self):
   for numPredictions in NUM_PREDICTIONS:
     runners.append(Runner(numPredictions))
 
-  for i in xrange(100000000):
+  for i in iter(int, 1):
```

Comment:
Oh oops, I accidentally included other unrelated changes into this PR. Will clean up before merging.